docs/deploy.md

```diff
@@ -36,20 +36,15 @@ To deploy cf-for-k8s as is, the cluster should:
 - be running Kubernetes version 1.14.x, 1.15.x, or 1.16.x
 - have a minimum of 5 nodes
 - have a minimum of 3 CPU, 7.5GB memory per node
-- have a metrics-server
-  - This may be as simple as running something like `kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.3.6/components.yaml`
-
-### IaaS Requirements
```

Comment:
hmm, I actually liked this section and the formatting it had before

Reply:
@jamespollard8 
The cluster requirements are the same as IaaS requirements, hence I collapsed them into one. In fact, there is no such thing as IaaS, it's just does your cluster provider support these requirements. 

The metrics server will go away once the metrics team will package it with cf-for-k8s.